packages/ts-transform/src/utils/css-transform.tsx

```diff
@@ -0,0 +1,89 @@
+import postcss, { plugin } from 'postcss';
+import autoprefixer from 'autoprefixer';
+import cssnano from 'cssnano-preset-default';
+import nested from 'postcss-nested';
+import whitespace from 'postcss-normalize-whitespace';
+
+const minify = () => {
+  const preset = cssnano();
+  // We exclude async because we need this to run synchronously as ts transformers aren't async!
+  const asyncPluginsToExclude = ['postcss-svgo'];
+
+  return preset.plugins
+    .map(([creator]: any) => {
+      // replicate the `initializePlugin` behavior from https://github.com/cssnano/cssnano/blob/a566cc5/packages/cssnano/src/index.js#L8
+      return creator();
+    })
+    .filter((plugin: any) => {
+      return !asyncPluginsToExclude.includes(plugin.postcssPlugin);
+    });
+};
+
+const parentOrphenedPseudos = plugin('parent-orphened-pseudos', () => {
+  return root => {
+    root.walkRules(rule => {
+      if (rule.selector.includes(':')) {
+        const newSelector = rule.selector
+          .split(',')
```

Comment:
this will fail in edge cases like `[data-foo=","]`

Reply:
thanks for the catch

Reply:
this is, of course, a hypothetical edge case - probably nobody would actually bump into this in their code 😅 but my general rule of thumb is to try to avoid this kind of things so I just don't have to deal with any potential issues. For dealing with contextualized strings things like `.split` or using regexps are always red flags for me to notice such possible edge cases

https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags/1732454#1732454